backend/src/main/java/gov/cdc/usds/simplereport/db/repository/PersonRepository.java

```diff
@@ -19,9 +19,18 @@
     @Query(BASE_QUERY + " and organization = :org")
     public List<Person> findAllByOrganization(Organization org, Sort sortBy);
 
+    @Query("from #{#entityName} e where e.organization = :org")
+    public List<Person> findAllByOrganizationIncludeDeleted(Organization org, Sort sortBy);
+
     @Query(BASE_QUERY + " and internalId = :id and organization = :org")
     public Optional<Person> findByIDAndOrganization(UUID id, Organization org);
 
+    @Query("from #{#entityName} e where e.internalId = :id and e.organization = :org")
+    public Optional<Person> findByIDAndOrganizationIncludeDeleted(UUID id, Organization org);
+
     @Query(BASE_QUERY + " AND e.organization = :org AND (e.facility IS NULL OR e.facility = :fac)")
     public List<Person> findByFacilityAndOrganization(Facility fac, Organization org, Sort sortBy);
+
+    @Query("from #{#entityName} e where e.organization = :org AND (e.facility IS NULL OR e.facility = :fac)")
+    public List<Person> findByFacilityAndOrganizationIncludeDeleted(Facility fac, Organization org, Sort sortBy);
```

Comment:
I would probably make the base query for the IncludeDeleted methods into a constant of its own, just to call attention to the similarity between them, but mostly my reaction here is that we are plainly reaching the limits of the naive Spring Data repository approach, and we should plan to rip all of this out when we implement search/filtering.